pkg/templates/aws/resources/security_group_rule.yaml

```diff
@@ -22,7 +22,7 @@ properties:
       security group rule
   SecurityGroupId:
     type: string # Not resource type because we need deploy time properties to be able to exist here
-    default_value: '{{ upstream "aws:security_group" .Self }}#Id'
+    required: true
```

Comment:
Should this also be namespaced to the SG?

Reply:
this has to be a string value so i dont think namespace fits too too well since we expect it to be a resource. And in some cases the SG wont exist in our graph either